ontology/uco/observable/observable.ttl

```diff
@@ -5753,6 +6180,32 @@ observable:TCPConnectionFacet
 	sh:targetClass observable:TCPConnectionFacet ;
 	.
 
+observable:Tablet
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf
+		observable:Computer ,
```

Comment:
And here is one reason we need to be careful with the name `observable:Computer` - the definition on `Computer` currently aligns that concept with form factors that would be found on or near a desk,.  A tablet is not appropriately a subclass of a class bearing that description, but it is appropriately a subclass of a more general "Computer".  `observable:Computer` should be distinguished from `observable:Device`.

Reply:
I believe this is resolved.